server/routerlicious/packages/lambdas/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@fluidframework/common-utils": "^0.27.0",
     "@fluidframework/gitresources": "^0.1020.0",
-    "@fluidframework/protocol-base": "^0.1020.0",
+    "@fluidframework/protocol-base": "*",
```

Comment:
this isn't right, as we should only reference * or -0 for unreleased packages, which means we need to release the dependency's  package before we release this package. if there is no -0 the package depends on a previously released version, so the dependency's  package doesn't need to be released.